src/main/java/com/sfuentes/Store.java

```diff
@@ -0,0 +1,24 @@
+package com.sfuentes;
+
+public class Store {
+
+  String item;
+  double price;
+  String color;
+
+  public Store(String item, double price, String color) {
+    this.item = item;
+    this.price = price;
+    this.color = color;
+  }
+
+  public double getPrice() {
+    double totalPrice = 0;
+    switch (color) {
```

Comment:
This will fail if you pass RED or red. Change it to be case insensitive and add a test for it.

Reply:
fixed.